src/Components/Blazor/Build/src/Microsoft.AspNetCore.Blazor.Build.csproj

```diff
@@ -1,9 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
*Things to talk to SDK:* This seems to work just fine. Do we need to cross-compile?

Reply:
This should be fine now.